beta/src/pages/learn/passing-data-deeply-with-context.md

```diff
@@ -195,7 +210,21 @@ You can't do it with props alone. This is where context comes into play. You wil
 
 Context lets a parent--even a distant one!--provide some data to the entire tree inside of it.
 
-<img alt="Context provides data to the entire tree" src="/images/docs/sketches/s_providing-context.png" />
+<DiagramGroup>
+
+<Diagram name="passing_data_context_close" height={430} width={608} alt="Diagram with a tree of three components. The parent contains a bubble representing a value highlighted in orange which projects down to the two children, each highlighted in orange.q" >
+
+Using context in close children
+
+</Diagram>
+
+<Diagram name="passing_data_context_far" height={430} width={608} alt="Diagram with a tree of ten nodes, each node with two children or less. The root parent note contains a bubble representing a value highlighted in orange. The value projects down directly to four leaves and one intermediate component in the tree, which are all highlighted in orange. None of the other intermediate components are highlighted.">
```

Comment:
```suggestion
<Diagram name="passing_data_context_far" height={430} width={608} alt="Diagram with a tree of ten nodes, each node with two children or less. The root node contains a bubble representing a value highlighted in orange. The value projects down directly to four leaves and one intermediate component in the tree, which are all highlighted in orange. None of the other intermediate components are highlighted.">
```

Do you think "root node" is sufficient? If we talk about "the root" (i.e. singular root) in a tree with multiple nodes then that node must also be a parent, no?